packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_page.dart

```diff
@@ -100,10 +100,23 @@ class _UpdatesOtherSoftwarePageState extends State<UpdatesOtherSoftwarePage> {
           ),
         ],
       ),
-      footer: model.onBattery
-          ? Html(
-              data: lang.onBatteryWarning(
-                  Theme.of(context).colorScheme.error.toHex()),
+      snackBar: model.onBattery
+          ? SnackBar(
+              width: 500,
```

Comment:
did you try what happens if the text is too long to fit? does it wrap?

Reply:
Yes, it wraps
![image](https://user-images.githubusercontent.com/113362648/220938985-1de59bf1-35b9-4594-9da6-69adc8ace139.png)